development/Dockerfile

```diff
@@ -18,9 +17,9 @@ RUN pip install --upgrade pip\
 # -------------------------------------------------------------------------------------
 # Remove redis==3.4.1 from the requirements.txt file as a workaround to #4910
 # https://github.com/netbox-community/netbox/issues/4910, required for version 2.8.8 and earlier
+ARG netbox_ver=master
```

Comment:
I recommend using `ARG NETBOX_VER = "${NETBOX_VER}"` then make sure the ENV includes NETBOX_VER from tasks.py.  Then you can get rid of your `--build-args` to docker.  This just helps consolidate how variables are handled in docker, the build, and with the configuration of netbox.